cmake/ExternalInclude.cmake

```diff
@@ -0,0 +1,19 @@
+set(RADIUM_EXTERNAL_CMAKE_OPTIONS
+    -DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}
+    -DCMAKE_CXX_STANDARD_REQUIRED=${CMAKE_CXX_STANDARD_REQUIRED}
+    -DCMAKE_GENERATOR=${CMAKE_GENERATOR}
+    -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+    -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
+    -DCMAKE_INSTALL_MESSAGE=${RADIUM_EXTERNAL_CMAKE_INSTALL_MESSAGE}
```

Comment:
At the first run on any external folder, this makes cmake to emit a warning message : 
```
CMake Warning:
    Manually-specified variables were not used by the project:
       CMAKE_INSTALL_MESSAGE
```

Reply:
Yes, but there is no typo and this option exists in the doc. It is however rarely used and maybe it’s implementation is not available on all version (the option has been introduced in cmake 3.1, and is still available).
So I suggest to keep it.

Reply:
I do not ask to remove `CMAKE_INSTALL_MESSAGE `but to find a way to silent this Warning. 
Having something in red that appears in a small log can be very distuubing if one think that everything goes well. And more, if this is not related to the user (but to the developer of the buildchain), why to perturb the user with such messages ? Perhaps there is a cmake policy that silent this ...